acnd.conf.patch by Jan Was <janek.jan@gmail.com>

Fix socket and pidfile paths in default config.

diff -urNad conf/acng.conf conf/acng.conf~
--- conf/acng.conf	2012-07-23 21:39:51.000000000 +0200
+++ conf/acng.conf~	2012-11-04 20:28:31.558838134 +0100
@@ -56,7 +56,7 @@
 
 # Socket file for accessing through local UNIX socket instead of TCP/IP. Can be
 # used with inetd bridge or cron client.
-# SocketPath:/var/run/apt-cacher-ng/socket
+SocketPath:/run/apt-cacher-ng/socket
 
 # Forces log file to be written to disk after every line when set to 1. Default
 # is 0, buffers are flushed when the client disconnects.
@@ -73,7 +73,7 @@
 # ForeGround: 0
 
 # Store the pid of the daemon process therein
-# PidFile: /var/run/apt-cacher-ng/pid
+PidFile: /run/apt-cacher-ng/pid
 
 # Forbid outgoing connections, work around them or respond with 503 error
 # offlinemode:0

